src/components/NotificationsModal/NotificationUserList/PureUserList.tsx

```diff
@@ -0,0 +1,51 @@
+import { PureUserListFragment$key } from '__generated__/PureUserListFragment.graphql';
```

Comment:
Fast follow to this landing is to consolidate this with the `FollowListUsers` component that I only noticed like 10 minutes ago